components/Card.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+const CardElement = styled.div`
+  padding: 1em 3em;
+  margin: 15px 0px;
+  text-align: center;
+  border: 3px grey solid;
+  border-radius: 10px;
+`;
+
+const StyledLink = styled.a`
+  text-decoration: none;
+`;
```

Comment:
you should use the Link component from next instead (don't forget to import it in your file)

```suggestion
const StyledLink = styled(Link)`
  text-decoration: none;
`;
```